.gitignore

```diff
@@ -41,4 +41,4 @@ __debug_bin
 terraform/downloaded*
 config.hcl
 terraform/tfplan.binary
-cq-provider
+cq-provider-gcp
```

Comment:
```suggestion
cq-provider
```

Is this change needed? `make run` creates a `cq-provider` binary

Reply:
yeah but go build creates `cq-provider-gcp` maybe we need both ?

Reply:
added both.